### PR TITLE
UI with empty initial properties. Fix:#1543 (partially)

### PR DESCRIPTION
**Pull request by oliviermgx**
Fixing issue #1543 (partially)
All properties managed by Silex in right panel will be set to empty at initial state.
Property general>display in right panel generated by grapesjs is removed (editor.StyleManager.removeProperty) then add (editor.StyleManager.addProperty) for test purpose, waiting decision with Grapesjs team to follow this policy or not.
All others properties managed by Grapesjs are non modified waiting Grapesjs decision.

### CODE REVIEW

File: src/ts/client/grapesjs/css-props.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default (editor: Editor, opts) => {
       }, {
         name: 'Overflow Y',
         type: 'radio',
-        options: [
+        options: [
```
Comment (lexoyo) [inline src/ts/client/grapesjs/css-props.ts:62]: ah la tu ajoutes des espaces pour rien

File: src/ts/client/grapesjs/css-props.ts [hunk 2]
```diff
@@ -355,8 +367,8 @@ export default (editor: Editor, opts) => {
       name: 'Word wrap',
       property: 'word-wrap',
       type: 'radio',
-      defaults: 'normal',
-      options: [
+      defaults: '',
+      options: [
```
Comment (lexoyo) [inline src/ts/client/grapesjs/css-props.ts:371]: same here, useless spaces

**Review by lexoyo — CHANGES_REQUESTED** [review]
It looks good to me :fireworks: 
Just 2 spaces to remove
Also do you check with `npm run lint` before you make a PR? That's a good practice
Thank you !!!

**Comment by oliviermgx** [comment]
Thank you for your comments and help !

**Comment by oliviermgx** [comment]
it's seems that npm run lint has been tested during automated Github check process ? 
Run with no pb on my side.

**Review by lexoyo — APPROVED** [review]

**Comment by lexoyo** [comment]
Aweaome 🙌🙌🙌
Fix #1543

**Pull request merged**